site/src/modules/dashboard/Navbar/NavbarView.tsx

```diff
@@ -174,18 +136,20 @@ export const NavbarView: FC<NavbarViewProps> = ({
           )}
         </NavLink>
 
-        <NavItems
-          css={styles.desktopNavItems}
-          canViewAuditLog={canViewAuditLog}
-          canViewDeployment={canViewDeployment}
-          canViewAllUsers={canViewAllUsers}
-          canViewHealth={canViewHealth}
-        />
+        <NavItems css={styles.desktopNavItems} />
 
         <div css={styles.navMenus}>
           {proxyContextValue && (
             <ProxyMenu proxyContextValue={proxyContextValue} />
           )}
+
+          <DeploymentDropdown
+            canViewAuditLog={canViewAuditLog}
+            canViewDeployment={canViewDeployment}
+            canViewAllUsers={canViewAllUsers}
+            canViewHealth={canViewHealth}
+          />
```

Comment:
I wonder if we can pass the root permissions to the component instead of creating derived values for each option.

Reply:
eh, I don't mind this. the root permissions names are a lot less clear imo, so having the `Navbar` "controller" abstract them with better names is nice.